policyengine_us/parameters/gov/states/ar/tax/income/deductions/itemized/amount/technical_institutes.yaml

```diff
@@ -0,0 +1,17 @@
+description: Arkansas defines the weighted average tuition for technical institutes at this amount under the post secondary education tuition deduction.
+values:
+  2021-01-01: 780
+
+metadata:
+  period: year
+  unit: currency-USD
+  label: Arkansas post secondary eduction tuition deduction technical institutes weighted average tuition 
+  reference:
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions # page 21
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=21
```

Comment:
this reference does not show the parameter value - $780 - please check all reference and remove the ones that do not point directly to parameter values